### PR TITLE
feat(panel): Allows centering to be used with animation transforms.

### PR DESCRIPTION
**Pull request by ErinCoughlan**
Fixes centering and built-in animation transforms by combining them. 

@jelbourn @ThomasBurleson @KarenParker @gmoothart @DerekLouie Please Review.

Note: Does NOT fix custom animations overwriting the panel centering transforms. That nasty matrix math will come in a future PR.

### CODE REVIEW

**Comment by KarenParker** [comment]
Are there any tests that are possible for this?

**Comment by ErinCoughlan** [comment]
Not that I know of. If you have any ideas for tests, I'll add them. As of right now, there are no animation tests.

**Comment by KarenParker** [comment]
LGTM

**Comment by jelbourn** [comment]
LGTM

File: src/components/panel/panel.js [hunk 1]
```diff
@@ -1128,46 +1128,35 @@ MdPanelRef.prototype._createPanel = function() {
...
-    // height and width.
-    return self._$rootScope['$$postDigest'](function () {
+    // correctly. This is necessary so that the panel will have a defined height
+    // and width.
+    self._$rootScope['$$postDigest'](function() {
       self._panelEl.css('top', positionConfig.getTop(self._panelEl));
       self._panelEl.css('bottom', positionConfig.getBottom(self._panelEl));
       self._panelEl.css('left', positionConfig.getLeft(self._panelEl));
       self._panelEl.css('right', positionConfig.getRight(self._panelEl));
       self._panelEl.css('transform', positionConfig.getTransform());
```
Comment (ThomasBurleson) [inline src/components/panel/panel.js]: Need modernizer prefixes here!

Comment (DevVersion) [inline src/components/panel/panel.js]: I think, using the `$mdConstant` is doing the same.

- https://github.com/angular/material/blob/master/src/core/util/constant.js#L39

Comment (ThomasBurleson) [inline src/components/panel/panel.js]: @DevVersion  - I had forgotten about that one. 👍 

> @DevVersion  please fix `vendorProperty( )` to sniff for **`-ms`** IE prefixes

**Pull request merged**